app/views/secrets.html

```diff
@@ -73,6 +73,31 @@
                   </tbody>
                 </table>
               </div>
+              <div ng-if="secretsByType.other.length !== 0">
+                <h3>Other Secrets</h3>
+                <table class="table table-bordered table-hover table-mobile secrets-table">
+                  <thead>
+                    <tr>
+                      <th>Name</th>
+                      <th>Type</th>
+                      <th>Created</th>
+                    </tr>
+                  </thead>
+                  <tbody ng-repeat="secret in secretsByType.other | orderBy : 'name'">
```

Comment:
Should this be `'metadata.name'`? The order in the screenshot looks right though.